test/jest/__mock__/stripesCore.mock.js

```diff
@@ -12,6 +12,13 @@ jest.mock('@folio/stripes/core', () => {
     IfPermission: jest.fn(({ perm, children }) => {
       return perm === 'permission' ? children : null;
     }),
+    coreEvents: {
```

Comment:
these values already exist in this import, do you really neeed mock it here? 

import { coreEvents } from '@folio/stripes/core';